### PR TITLE
feat: Cloud Gateway terminology updates

### PR DESCRIPTION
**Pull request by cloudjumpercat**
### Description

We are now calling the original way of deploying ‘self-managed’, and cloud gateways are now referred to as fully-managed, or dedicated cloud instances. Update the Konnect documentation to ensure that we are referring to self-managed control planes across the docs.

I manually went through the docs to look for spots to update since there weren't any great search terms to find/replace in the docs set.
 

DOCU-3701
### Testing instructions

Preview link: https://deploy-preview-7093--kongdocs.netlify.app/konnect/

### Checklist 

- [x] Review label added 
- [x] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

For example, if this change is for an upcoming 3.6 release, enclose your content in `{% if_version gte:3.6.x %} <content> {% endif_version %}` tags (or `if_plugin_version` tags for plugins). 

Use any of the following keys:
* `gte:<version>` - greater than or equal to a specific version
* `lte:<version>` - less than or equal to a specific version
* `eq:<version>` - exactly equal to a specific version

You can do the same for older versions.

### CODE REVIEW

File: app/konnect/getting-started/index.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ title: Get started with Konnect
 
 ## About {{site.konnect_short_name}}
 
-{{site.konnect_short_name}} is an API lifecycle management platform that lets you build modern applications better, faster, and more securely. The control plane is hosted in the cloud by Kong, while the data plane is managed by you within your preferred network environment. All of this is powered by {{site.base_gateway}} — Kong's lightweight, fast, and flexible API gateway. 
+{{site.konnect_short_name}} is an API lifecycle management platform that lets you build modern applications better, faster, and more securely. The control plane is hosted in the cloud by Kong, while the data plane is managed by you or Kong within your preferred network environment. All of this is powered by {{site.base_gateway}} — Kong's lightweight, fast, and flexible API gateway.
```
Comment (Guaris) [inline app/konnect/getting-started/index.md:31]:
```suggestion
{{site.konnect_short_name}} streamlines the API lifecycle management process, empowering you to develop modern applications with enhanced speed, improved security, and greater efficiency. It uniquely combines a control plane, managed by Kong and hosted in the cloud, with the versatility of managing the data plane on your terms — either self-managed or through Kong, within your preferred network environment. At the heart of this platform is {{site.base_gateway}}, Kong's innovative, high-performance, and adaptable API gateway, designed to meet your application development needs with agility and reliability.
```

File: app/konnect/index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ breadcrumb: Overview
 management platform designed from the ground up for the cloud native era
 and delivered as a service. This platform lets you build modern applications
 better, faster, and more securely. The control plane is hosted
-in the cloud by Kong, while the data plane is managed by you within your
+in the cloud by Kong, while the data plane is managed by you or Kong in your
```
Comment (Guaris) [inline app/konnect/index.md:13]: Same as in the doc above. I don't think its powerful to say managed by you 'or kong'. I think highlighting choice and flexibility is the most important part.

File: app/konnect/index.md [hunk 3]
```diff
@@ -79,7 +79,7 @@ across the organization’s applications.
 collaborate and manage their own set of data plane nodes and services without
 the risk of impacting other teams and projects. Gateway Manager instantly
 provisions a hosted {{site.base_gateway}} control plane and supports securely
-attaching data plane nodes from your cloud or hybrid environments.
+attaching data plane nodes from your fully-managed cloud, or self-managed cloud and hybrid environments.
```
Comment (Guaris) [inline app/konnect/index.md:82]:
```suggestion
attaching self-hosted, or fully-managed data plane nodes and hybrid environments.
```

File: app/konnect/gateway-manager/data-plane-nodes/index.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ Advanced setup:
 {:.note}
 > **Notes:** 
 > * Gateway Manager includes a feature called Control Plane Launcher which can be used with any of AWS, Azure, or GCP. This feature is currently in tech preview.
-> * Kong does not host data plane nodes. You must install and host your own.
+> * Kong only hosts fully-managed Dedicated Cloud Gateway data plane nodes. You must install and host your own if you want to use a self-managed data plane.
```
Comment (Guaris) [inline app/konnect/gateway-manager/data-plane-nodes/index.md:39]: We host both control planes and data planes now so the word 'only' here throws me off. Maybe "Kong can host data planes with our Dedicated Cloud Gateways offering, if you want to use a self-managed data plane, you must and install and host your own"

File: app/konnect/gateway-manager/data-plane-nodes/upgrade.md [hunk 5]
```diff
@@ -1,16 +1,19 @@
 ---
-title: Upgrade a Data Plane Node to a New Version
+title: Upgrade a Self-managed Data Plane Node to a New Version
```
Comment (Guaris) [inline app/konnect/gateway-manager/data-plane-nodes/upgrade.md:2]:
```suggestion
title: Upgrade a Data Plane Node to a New Version
```

Maybe not here so we can write about upgrading both types of DPs

File: app/konnect/gateway-manager/data-plane-nodes/upgrade.md [hunk 6]
```diff
@@ -1,16 +1,19 @@
...
-You can upgrade data plane nodes to a new {{site.base_gateway}} version by bringing
+You can upgrade self-managed data plane nodes to a new {{site.base_gateway}} version by bringing
 up new data plane nodes, and then shutting down the old ones. This is the best
 method for high availability, as the new node starts processing data before the
 old node is removed. It is the cleanest and safest way to upgrade with no
 proxy downtime.
 
 We recommend running one major version (2.x or 3.x) of a data plane node per control plane, unless you are in the middle of version upgrades to the data plane. Mixing versions may cause [compatibility issues](/konnect/gateway-manager/version-compatibility).
 
+{:.note}
```
Comment (Guaris) [inline app/konnect/gateway-manager/data-plane-nodes/upgrade.md:14]: Great

**Review by Guaris — CHANGES_REQUESTED** [review]

File: app/konnect/gateway-manager/data-plane-nodes/index.md (hunk 4 above)
Comment (lena-larionova) [reply app/konnect/gateway-manager/data-plane-nodes/index.md:39]: I think we can remove this note altogether. CGWs _are_ the hosted offering.

File: app/konnect/gateway-manager/index.md [hunk 7]
```diff
@@ -149,11 +149,11 @@ You cannot delete the default control plane.
 A data plane node is a single {{site.base_gateway}} instance. 
 Data plane nodes service traffic for the control plane. 
 
-Kong does not host data plane nodes.
-You must deploy your own nodes, either on your own systems or in 
+Kong only hosts fully-managed Dedicated Cloud Gateway data plane nodes.
+You must deploy your own nodes if you want to self-manage them, either on your own systems or in 
 an external cloud provider.
```
Comment (lena-larionova) [inline app/konnect/gateway-manager/index.md:154]:
```suggestion
You can have deploy your data plane nodes in one of the following ways:
* Self-managed nodes, hosted either on your own systems or in 
an external cloud provider
* Fully-managed dedicated cloud gateway nodes
```
Rephrasing to make this sound more positive. The previous phrasing implied a lack of something.

File: app/konnect/index.md (hunk 3 above)
Comment (lena-larionova) [reply app/konnect/index.md:82]: Suggestion off of Angel's suggestion, as the "hybrid environments" portion of the sentence wasn't in the right place.

```suggestion
attaching self-hosted nodes running in cloud or hybrid environments, or using fully-managed data plane nodes.
```
I'm still not 100% on this phrasing, but the point I'm trying to get to it - self-hosted is what runs anywhere, fully-managed doesn't have that control.

**Review by Guaris — APPROVED** [review]

**Pull request merged**